errors/app.errors.js

```diff
@@ -2,20 +2,20 @@ exports.handleCustomErrors = (err, req, res, next) => {
     if (err.status && err.msg) {
         res.status(err.status).send({ msg: err.msg });
     } else next(err);
-}
+};
 
 exports.handlePsqlErrors = (err, req, res, next) => {
     if (err.code === "23503") {
-        res.status(400).send({ msg: "invalid input"})
+        res.status(400).send({ msg: "invalid input" });
     } else if (err.code === "23502") {
-        res.status(400).send({ msg: "invalid input"})
+        res.status(400).send({ msg: "invalid input" });
     } else if (err.code === "22P02") {
         res.status(400).send({ msg: "invalid input" });
```

Comment:
Would be better if the error messages are different, it'll help with frontend debugging
e.g. 22P02 - invalid input
23502 - missing required key etc.